src/components/ArticlePage.jsx

```diff
@@ -51,14 +69,32 @@ const ArticlePage = () => {
         </Row>
         <Row>
           <Col sm={3}></Col>
-          <Col sm>
-            <button>UpCroak</button>
-          </Col>
-          <Col sm>
-            <p>{article.votes}</p>
-          </Col>
-          <Col sm>
-            <button>DownCroak</button>
+          <Col>
+            <ButtonGroup className="button-group">
+              <Button
+                disabled={isLoadingUp}
+                onClick={
+                  !isLoadingUp
+                    ? () => {
+                        handleCroak("up");
```

Comment:
If instead of 'up' here you passed 1, and in the one below you passed -1, then you could just have patchArticleVotes called once in the handleCroak function - just pass it whatever value you've given it.